### ISSUE TITLE
Handle errors when loading `next.config.js`

### ISSUE THREAD

**Issue by ehmicky** [netlify/netlify-plugin-nextjs#79]
There are many reasons why loading `next.config.js` might fail such as:
  - Invalid configuration value ([example](https://app.bugsnag.com/netlify/netlify-build/errors/5fdf345b5c974300192886ec))
  - Syntax error ([example](https://app.bugsnag.com/netlify/netlify-build/errors/5ff228c670d83600174cac05))
  - General error in the JavaScript logic, throwing an exception ([example](https://app.bugsnag.com/netlify/netlify-build/errors/5fe1a057808a0e0018bce5cd))

Those errors are currently reported as plugin bugs. We should report those as user errors instead by adding a `try`/`catch` + `utils.build.failBuild()` block.

**Comment by ehmicky** [comment]
Other example [here](https://app.bugsnag.com/netlify/netlify-build/errors/60073abf129fb70017badeed):

```
Specified i18n.defaultLocale should be included in i18n.locales.
See more info here: https://err.sh/next.js/invalid-i18n-config 
```

**Comment by ehmicky** [comment]
This bug was fixed in #99 for: 

https://github.com/netlify/netlify-plugin-nextjs/blob/61b61f57d58ae095e3522c86013c75823911179b/helpers/hasCorrectNextConfig.js#L17

However, the configuration is loaded also here:

https://github.com/netlify/netlify-plugin-nextjs/blob/61b61f57d58ae095e3522c86013c75823911179b/src/lib/helpers/getNextConfig.js#L11

We should use a similar `try`/`catch`/`failBuild` block there as well.

[Bugsnag error](https://app.bugsnag.com/netlify/netlify-build/errors/604a23fdfea8af0007fb09f6)

**Comment by ehmicky** [comment]
First part at https://github.com/netlify/netlify-plugin-nextjs/pull/114

**Comment by ehmicky** [comment]
Fixed by #124.
